### PR TITLE
{bio}[foss/2022b] TF-COMB v1.1, TOBIAS v0.16.1, svist4get v1.3.1, ...

### PR DESCRIPTION
**Pull request by pavelToman**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by pavelToman** [comment]
@boegelbot please test @ generoso

**Comment by pavelToman** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in total)
node3155.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/boegel/32ea94cf283c37892415d10acfa6aa18 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen3

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in total)
node3155.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/boegel/e96a8c3f511c8f699b6e0eb756b1301f for a full test report.

File: easybuild/easyconfigs/t/TF-COMB/TF-COMB-1.1-foss-2022b.eb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    (name, version, {
+        'preinstallopts': local_tfcomb_preinstallopts,
+        'modulename': 'tfcomb',
+        'checksums': ['5b718061660e0f9f94d86459eb742ca81de5851b0defd8b08c8a7a7e3370c253'],
+    }),
+]
+
+sanity_check_commands = [
+    "python -c 'import gv'",
+]
```
Comment (boegel) [inline easybuild/easyconfigs/t/TF-COMB/TF-COMB-1.1-foss-2022b.eb:50]: this part doesn't make sense for this easyconfig, probably a copy-paste error?
```suggestion
```

**Review by boegel — CHANGES_REQUESTED** [review]

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
Going in, thanks @pavelToman!

**Pull request merged**